src/configs/config-fonts.json

```
@@ -0,0 +1,35 @@
+{
+  "source": ["src/tokens/design-tokens.json"],
+  "platforms": {
+    "css": {
+      "transforms": [
+        "attribute/cti",
+        "name/cti/kebab",
+        "time/seconds",
+        "content/icon",
+        "size/pxToRem",
+        "color/css"
+      ],
+      "buildPath": "build/css/",
+      "files": [
+        {
+          "destination": "font-faces.css",
+          "format": "css/font-faces",
+          "options": {
+            "outputReferences": true
+          }
+        }
+      ]
+    },
+    "js": {
+      "transforms": ["attribute/cti", "name/cti/kebab"],
+      "buildPath": "build/ts/",
+      "files": [
+        {
+          "destination": "preloads.ts",
```

vu qu'il n'y a que des fonts, peut etre renommer tout les "preloads" en "font preload" ?

```suggestion
          "destination": "font-preloads.ts",
```

---

Ouais, c'est plus cohérent je suis d'accord, je modifie 👍